src/send/send_caliper_events.py

```diff
@@ -0,0 +1,69 @@
+import requests
+import fileinput
+import json
+import time
+
+
+def api_handler(data):
+    AUTHORIZATION = 'Authorization'
+    BASIC = 'Basic '
+    MIME_TYPE_JSON = 'application/json'
+    CONTENT_TYPE = 'Content-type'
+    headers = {CONTENT_TYPE: MIME_TYPE_JSON, AUTHORIZATION: BASIC + TOKEN}
+    response = None
+    try:
+        response = requests.post(url, data=data, headers=headers)
```

Comment:
By default, does this reuse connections to the server that have been opened earlier?  If not, could it be made to reuse connections?  That could help improve processing time.  Reusing connections would require checking the old connection before use and opening a new one if the old one isn't working.

That said, I'm not certain OpenLRS or the AWS environment will be happy with sending multiple HTTP POSTs over a single connection.  It probably is, because I believe that's the common way web browsers and servers operate.  It would be worth testing.